schematic/store/synapse.py

```diff
@@ -2825,7 +2825,7 @@ def getDatasetAnnotations(
             try:
                 logger.info("Trying batch mode for retrieving Synapse annotations")
                 table = self.getDatasetAnnotationsBatch(datasetId, dataset_file_ids)
-            except (SynapseAuthenticationError, SynapseHTTPError):
+            except (SynapseAuthenticationError, SynapseHTTPError, ValueError):
```

Comment:
Thanks for adding this.  Not to add to your work, but we may want a unit test to mock that this logic is working as expected to catch the different exceptions.

Let me know if you want to meet to discuss how to add a test specifically for `getDatasetAnnotations` function

It's a bit more work, but if you mock the particular calls, then we aren't actually using any test projects.